.gitignore

```diff
@@ -1,2 +1,7 @@
+.DS_Store
```

Comment:
For any python project I would just start with https://github.com/github/gitignore/blob/master/Python.gitignore and add whatever else you need.

Reply:
Great, I will use this from now on